app/routes/news._index/route.mdx

```diff
@@ -27,6 +27,28 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2025
 
+    <CollectionItem
+      className="maxw-none grid-container"
+      variantComponent={
+        <CollectionCalendarDate datetime={'April 14, 2025'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Circulars Archive Viewable by Event </Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        The Circulars archive now provides two options for viewing and searching:
+          - Circulars View
```

Comment:
Instead of a doubly-nested unordered list, for more typographical emphasis, make `Circulars View` and `Events View` two subheadings.